cves/kernel/CVE-2020-12770.yml

```diff
@@ -105,10 +107,10 @@ vcc_instructions: |
 
   Place any notes you would like to make in the notes field.
 vccs:
-- commit: "1da177e4c3f41524e886b7f1b8a0c1fc7321cac2"
-  note: Discovered automatically by archeogit.
-- commit: a64e5a868573d6fe3b76e8d17538b10499239631
-  note: Discovered automatically by archeogit.
+- commit: 1da177e4c3f41524e886b7f1b8a0c1fc7321cac2
+  note: Manually Discovered
+- commit: 
+  note: 
 upvotes_instructions: |
```

Comment:
I would upvote this as a 3.

Reply:
I would agree and place it at a 3.

Reply:
I will give this vulnerability 3 upvotes!